website/docs/guides/1.8.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-180-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.8.0 has a number of new and exciting features and changes.
+
+**Major Enhancements:**
+* The HashiCorp Terraform MongoDB Atlas Provider now supports [Snapshot Distribution](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule) 
+  * To learn more about this feature see here: https://www.mongodb.com/blog/post/introducing-snapshot-distribution-atlas   
+* The HashiCorp Terraform MongoDB Atlas Provider now allows users to [Programmatically Create API Keys](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/api_key) 
+  * To learn more about this feature see here: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Programmatic-API-Keys
+  *  **IMPORTANT WARNING:**  Creating, Reading, Updating, or Deleting Atlas API Keys may expose sensitive organizational secrets to Terraform State. Consider storing sensitive API Key secrets instead via the [HashiCorp Vault MongoDB Atlas Secrets Engine](https://developer.hashicorp.com/vault/docs/secrets/mongodbatlas). 
+
+
+**Depreciations:**
+* [`cloud_provider_snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot), [`cloud_provider_snapshot_backup_policy`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_backup_policy), [`cloud_provider_snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_restore_job), and [`private_ip_mode`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/private_ip_mode) are now deprecated and will be removed from codebase as of v1.9
```

Comment:
I would make these two bullets.  I would also let the user know cloud_backup.... is the replacement for each of these, similar to the message we have in the docs for these.  I would also change the lang to make clear we deprecated these a while ago.   Make sure if you use the suggestion to replace >>> and <<<
```suggestion
* [`mongodbatlas_cloud_provider_snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot), [`mongodbatlas_cloud_provider_snapshot_backup_policy`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_backup_policy), [`mongodbatlas_cloud_provider_snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_restore_job) have been deprecated for a few versions.  We will proceed with removal in the next major version, 1.9.  They have been replaced by >>>add the 3 new ones in the same order as above<<<< respectively. 
* [`mongodbatlas_private_ip_mode`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/private_ip_mode) has been deprecated for a few versions.  We will proceed with removal in the next major version, 1.9.  
```

Reply:
updated